Products overhaul (#77)
pretix/pretix-docs: docs/guides/products/discounts.md

# Discounts 

pretix has several different ways for offering your customers discounts on their purchase under certain conditions. 
This article explains several use cases for different types of discounts. 

## Prerequisites 

Most of the methods described here are handled on the event level, so you have to create an event first. 
The article assumes some general knowledge on how to create and edit products, so it makes sense to take a look at the [guide on products](index.md) first. 

> ## How To 

Reviewer: ## How To 

Strukturell wirkt es aktuell ein bisschen unübersichtlich, dass unter How to direkt (d.h. ohne Zwischenebene) die einfache Variante beschrieben wird und alle anderen Varianten dann unter weiteren Unterpunkten.
1-2 Sätze Inhalt für How to würden ja reichen, um dann die einfache Variante auslagern zu können.

Set the capacity of the quota to the maximum number of tickets you want to sell. 

If you want to place an additional limit on the number of discount tickets available, create another quota and set the capacity to the maximum number of discount tickets you want to sell. 
Add only the discount tickets to this quota. 
pretix will subtract from both quotas whenever a discount ticket is sold, and only subtract from the general quota when a regular ticket is sold. 

The following sections will guide you through some more advanced methods for offering discounts based on certain conditions. 

### Early bird tickets

This section explains how to offer early bird tickets that can only be bought far in advance of the event and which get replaced with more expensive tickets at some point. 
It is possible to create a ticket and manually increase the price as the event approaches. 

But pretix also offers two methods for automating this: 
You can offer different pricing tiers based on date, or based on the number of tickets that are still available. 
Regardless of which method you use, the first step is creating one admission ticket for each price tier. 

> #### Early bird tickets based on dates for singular events

Reviewer: > #### Early bird tickets based on dates for singular events

Die Formulierung der Überschrift führt dazu, dass ich dann eine Überschrift wie
> Early bird tickets based on dates for event series

suche, welche nicht existiert. 
Ergibt die Einschränkung auf singular events in der Überschrift so Sinn, vgl. weiter unten?

If you want to offer early bird tickets based on dates for a singular event, edit one of the products and switch to the :btn:Availability: tab. 
Use the "Available from" and "Available until" fields to define the period of time in which the product can be purchased. 
Repeat this step for each product. 

Make sure that the "Available until" option for the first ticket has the same date and time as the "Available from" option for the following ticket. 
This is to ensure that there is no overlap during which more than one pricing tier is available, and no gap during which no tickets are available. 

> The method described above does not work for an event series with dates repeating over a large span of time. 

Reviewer: > The method described above does not work for an event series with dates repeating over a large span of time. 

Die Überschrift beschränkt sich auf "singular events", warum geht es hier plötzlich um event series?

If you want to offer early bird tickets based on dates for a singular event, navigate to :navpath:Your event series → :fa3-calendar: Dates: and edit one of the dates. 
Under "Product settings", use the "Available from" and "Available until" fields to define the period of time in which the product can be purchased for this individual date. 

If you want to set the same availability for multiple dates, navigate to :navpath:Your event series → :fa3-calendar: Dates:, check the box next to each date you want to edit and click the :btn-icon:fa3-edit: Edit selected: button. 
Under "Item prices", use the "Available from" and "Available until" fields to define the period of time in which the product can be purchased for the selected dates. 
Check the boxes labeled "change" next to "Available from" and "Available until" to ensure that the product settings are overridden. 

#### Early bird tickets based on ticket numbers 

> If you want to offer early bird tickets based on ticket numbers for a singular event, create one quota for each price level. 

Reviewer: > If you want to offer early bird tickets based on ticket numbers for a singular event, create one quota for each price level. 

Dass hier die Rede von "singular event" ist, schafft die Erwartung, dass an anderer Stelle noch auf Veranstaltungsreihen eingegangen wird, was aber nicht der Fall ist.
War das Absicht oder ist das einfach noch mal der Einleitungssatz des vorigen Abschnitts?

> If you want to offer early bird tickets based on ticket numbers for a singular event, create one quota for each price level. 

Reviewer: > If you want to offer early bird tickets based on ticket numbers for a singular event, create one quota for each price level. 

Der Satz wiederholt relativ 1:1 die Überschrift und liefert dadurch in der ersten Hälfte wenig Informationen. Würde das leicht umformulieren, also sowas wie 
> ... based on the number of tickets already sold for a particular event

oder schöner.

 - "Early bird and lower" with a total capacity of 200 and both the "Super early bird ticket" and the "Early bird ticket" products selected. 
 - "All participants" with a total capacity of 400, all three products selected and no additional options.

Next, modify the product "Regular ticket" and switch to the :btn:Availability: tab.
Under "Only show after sellout of", select your quota "Early bird and lower". 
Do the same for the "Early bird ticket" with the quota "Super early bird ticket".

### Discount packages 

This section explains how to offer combinations of several products at lower prices than the sum of the prices of each individual product. 
This is useful whenever you are offering several products, expect customers to purchase different combinations of those products, and want to offer lower prices for larger combinations. 
You can apply this, for example, to any of the following use cases: 

 - a trade fair or festival opening on three consecutive days with different pricing for single-day, two-day, and three-day passes
 - different levels of merch packages with larger packages offered at reduced rates 

>  - different offers such as swimming, sauna, and spa, which are offered at a cheaper price if purchased in combination 

Author: Vorschlag: 
>  - different offers such as in-person workshops, online content, and a networking event, which are offered at a cheaper price if purchased in combination

There are two methods to implement this using pretix: 
One method uses combination products. 
The other uses bundles. 
Both methods will be explained in the following subsections. 

> Depending on your use case, it may also be appropriate to forgo these options and offer all products at full price. 

Reviewer: > Depending on your use case, it may also be appropriate to forgo these options and offer all products at full price. 

1. Eine kurze Beschreibung, in welcher Situation eher A oder eher B sinnvoll ist, wäre nützlich.
2. Die Aussage "Wenn du Rabatt geben willst, kann es besser sein, das einfach zu lassen und alles zum vollen Preis anzubieten" bedarf großflächiger Erklärung.

If you want to offer a discount for large orders regardless of which products are purchased, refer to the section on [group discounts](discounts.md#group-discounts) instead. 

#### Option A: Combination products

One option is to create the basic products and quotas, and to then create separate products for all possible combinations of basic products. 
This has the advantage of keeping your products and possible orders fairly straightforward. 
It can be implemented without touching the add-ons or bundles features. 

However, this method is only feasible if the total number of possible combinations is rather small. 
The number of products you need to offer in your shop is `n²-1`, where `n` is the number of basic products. 
This number grows exponentially with every additional basic product. 

> If you have three basic products, it results in seven products having to be offered in your shop. 

Reviewer: > If you have three basic products, it results in seven products having to be offered in your shop. 

3²-1 ist meiner Rechnung nach nicht 7, oder?

For illustrative purposes, assume you are hosting a three-day trade fair. 
Create a basic ticket for each day of the trade fair, three combination tickets for two days, and one combination ticket for all three days: 

 - Day 1 ticket
 - Day 2 ticket
 - Day 3 ticket
 - Day 1 + day 2 ticket  
 - Day 1 + day 3 ticket 
 - Day 2 + day 3 ticket
 - Ticket for all three days

Then, create three quotas, each one with a total capacity equal to your venue's maximum capacity on any given day:

 - Day 1 quota, linked to "Day 1 ticket", "Day 1 + day 2 ticket", "Day 1 + day 3 ticket", and "Ticket for all three days"
 - Day 2 quota, linked to "Day 2 ticket", "Day 1 + day 2 ticket", "Day 1 + day 2 ticket", and "Ticket for all three days"
 - Day 3 quota, linked to "Day 3 ticket", "Day 1 + day 3 ticket", "Day 2 + day 3 ticket", and "Ticket for all three days"

This way, every attendee can order exactly one ticket that they can use for all days that they are going to attend. 
Finally, navigate to :navpath:Your event → :fa3-check-square-o: Check-in:, edit your check-in list and switch to the :btn:Advanced: tab. 

> Define custom check-in rules so that the tickets in the first quota are only valid on the first day of the event; the tickets in the second quota are only valid on the second day; and the tickets in the third quota are only valid on the third day. 

Reviewer: > Define custom check-in rules so that the tickets in the first quota are only valid on the first day of the event; the tickets in the second quota are only valid on the second day; and the tickets in the third quota are only valid on the third day. 

Nachdem wenige Zeilen zuvor z.B. das "Day 1 + Day 2 ticket" mit der Day 1 quota verknüpft wurde, ist 
> **only** valid on the first day of the event

widersprüchlich (analog für die anderen Kontingente).

The logic looks like this: 

At least one of the conditions below (OR)

 - All of the conditions below (AND)
    - Product is one of "Day 1 ticket", "Day 1 + day 2 ticket", "Day 1 + day 3 ticket", or "Ticket for all three days"
    - Current day of the week = 1 (Monday)
 - All of the conditions below (AND)
    - Product is one of "Day 2 ticket", "Day 1 + day 2 ticket", "Day 1 + day 2 ticket", or "Ticket for all three days"
    - Current day of the week = 2 (Tuesday)
 - All of the conditions below (AND)
    - Product is one of "Day 3 ticket", "Day 1 + day 3 ticket", "Day 2 + day 3 ticket", or "Ticket for all three days"
    - Current day of the week = 3 (Wednesday)

![Custom check-in rule with three sets of AND-conditions. The first one demands that the product is one of the products for day one AND that the current day of the week is Monday. The second and third set do the same thing, but for day 2 and Tuesday as well as day 3 and Wednesday respectively. The three sets of AND-conditions are all included in one OR-condition.](../../assets/screens/products/check-in-rules-combination-products.png "Custom check-in rule") 

#### Option B: Add-ons and bundles

> Another option is to create the basic products and quotas, and to then create products with mandatory addons for all possible combinations. 

Reviewer: ```suggestion
Another option is to create the basic products and quotas, and to then create products with mandatory add-ons for all possible combinations. 
```

An exception can be made for the combination product containing **all** basic products. 
The full combination product can either be set up as described above in the section [Option A: Combination products](discounts.md#option-a-combination-products), or as a bundle. 
The approach using the bundle will also be described in this section. 

This option has the advantage that the number of products needed does not grow exponentially with the number of basic products. 

> The main disadvantage is that orders become more complex because ever order containing a product with mandatory addons will contain at least three products. 

Reviewer: ```suggestion
The main disadvantage is that orders become more complex because every order containing a product with mandatory add-ons will contain at least three products. 
```

For illustrative purposes, assume you are hosting a three-day trade fair. 
First, create a category for "Day tickets". 
Then, create a basic admission ticket for each day of the trade fair, one non-admission product for any two days, and one non-admission product for all three days: 

 - Day 1 ticket
 - Day 2 ticket
 - Day 3 ticket
 - Ticket for two days
 - Ticket for all three days

Add the tickets for day 1, 2 and 3 to the "Day tickets" category. 
Do **not** add any other tickets to that category. 

> Then, create one quota for each of the tickets. 

Reviewer: ```suggestion
Then, create one quota for each of the admission tickets. 
```

> When creating the quotas for each single day ticket, set the "Total capacity" to a number equal to the capacity of your venue for each one. 

Reviewer: ```suggestion
When creating the quotas for each single day ticket, set the "Total capacity" to a number equal to the capacity of your venue for each day.
```

The quotas for the two-day and three-day ticket can be unlimited. 

Edit the "Ticket for two days", switch to the :btn:Add-on: tab and click the :btn-icon:fa3-plus: Add a new add-on: button. 
Select the category "Day tickets", set the minimum and maximum number to 2 and check the box next to "Add-ons are included in the price". 
Click the :btn:Save: button. 

> With this configuration, a customer purchasing the "Ticket for two days" will be prompted to add two tickets from the "Day tickets" category to their purchase as addons. 

Reviewer: ```suggestion
With this configuration, a customer purchasing the "Ticket for two days" will be prompted to add two tickets from the "Day tickets" category to their purchase as add-ons. 
```

 - Day 3 ticket
 - Day 1 + day 2 ticket  
 - Day 1 + day 3 ticket 
 - Day 2 + day 3 ticket
 - Ticket for all three days

Then, create three quotas, each one with a total capacity equal to your venue's maximum capacity on any given day:

 - Day 1 quota, linked to "Day 1 ticket", "Day 1 + day 2 ticket", "Day 1 + day 3 ticket", and "Ticket for all three days"
 - Day 2 quota, linked to "Day 2 ticket", "Day 1 + day 2 ticket", "Day 1 + day 2 ticket", and "Ticket for all three days"
 - Day 3 quota, linked to "Day 3 ticket", "Day 1 + day 3 ticket", "Day 2 + day 3 ticket", and "Ticket for all three days"

This way, every attendee can order exactly one ticket that they can use for all days that they are going to attend. 
Finally, navigate to :navpath:Your event → :fa3-check-square-o: Check-in:, edit your check-in list and switch to the :btn:Advanced: tab. 
Define custom check-in rules so that the tickets in the first quota are valid on the first day of the event; the tickets in the second quota are valid on the second day; and the tickets in the third quota are valid on the third day. 

You can do this either using the "Current day of the week" or the "Current date and time" condition. 

> The check-in rule could look similar to the one in the screenshot below. 

Author: Der Artikel ist in Arbeit :) sollte aber sobald beides fertig ist, hier verlinkt werden. Ich lasse das erstmal so stehen
